exporting/send_data.c

```diff
@@ -132,52 +153,47 @@ void simple_connector_receive_response(int *sock, struct instance *instance)
  * @param failures the number of communication failures.
  * @param instance an instance data structure.
  */
-void simple_connector_send_buffer(int *sock, int *failures, struct instance *instance)
+void simple_connector_send_buffer(
+    int *sock, int *failures, struct instance *instance, BUFFER *header, BUFFER *buffer, size_t buffered_metrics)
 {
-    BUFFER *buffer = (BUFFER *)instance->buffer;
-    size_t len = buffer_strlen(buffer);
-
     int flags = 0;
 #ifdef MSG_NOSIGNAL
     flags += MSG_NOSIGNAL;
 #endif
 
 #ifdef ENABLE_HTTPS
     uint32_t options = (uint32_t)instance->config.options;
-    struct opentsdb_specific_data *connector_specific_data = instance->connector_specific_data;
+    struct simple_connector_data *connector_specific_data = instance->connector_specific_data;
 
     if (options & EXPORTING_OPTION_USE_TLS)
         ERR_clear_error();
 #endif
 
     struct stats *stats = &instance->stats;
-
-    int ret = 0;
-    if (instance->send_header)
-        ret = instance->send_header(sock, instance);
-
     ssize_t written = -1;
```

Comment:
It is not necessary to keep this initial value here, because it won't be used.

Reply:
Removed.